_includes/architecture.md

```diff
@@ -0,0 +1,21 @@
+
+![Blockstack Architecture](/common/images/architecture.png)
+
+Blockchains require consensus among large numbers of people, so they can be slow. Additionally, a blockchain is not designed to hold a lot of data. This means using a blockchain for every bit of data a user might write and store is expensive. For example, image if an application were storing every tweet in the chain.
+
+Blockstack addresses blockchain performance problems by taking a layered approach. At the base of the system is a blockchain and the Blockstack Naming System(BNS). The blockchain  governs ownership of names (identities)  in the system, names such as domain names, username, and applications names.
+
+Names in Blockstack correspond to routing data in the OSI stack. The routing data is stored in the Atlas Peer Network, the second layer. Every core node that joins the Blockstack Network is able to obtain an entire copy of this routing data. Blockstack uses the routing data to associate names (usernames, domains, application names) with a particular storage location.
```

Comment:
Should we write out "OSI" in full since it's the first usage on the page? I'm not personally sure what it is.

Reply:
Also "(usernames, domains, and application names)" with "and"

Reply:
OSI is the standard 7 layer model describing a network from physical to the application layer.  Network people pretty much know it by heart.  So, routing is a layer 3 activity.